acl_setup.php

```diff
@@ -236,7 +258,7 @@
  $gacl->add_acl(
   array(
    'acct'=>array('bill', 'disc', 'eob', 'rep', 'rep_a'),
-   'admin'=>array('calendar', 'database', 'forms', 'practice', 'superbill', 'users', 'batchcom', 'language', 'super', 'drugs', 'acl'),
+   'admin'=>array('calendar', 'database', 'forms', 'practice', 'superbill', 'users', 'batchcom', 'language', 'super', 'drugs', 'acl','multipledb','gadd','gcalendar','glog'),
```

Comment:
Note there are 5 standard user roles:
Administrator
Physician
Clinician (think of as a midlevel provider like a physician assistant or nurse)
Receptionist (Scheduler)
Accountant

Which roles do you think should get out of the box access to your new ACO's ?